core-bundle/src/Cron/Cron.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\CoreBundle\Entity\Cron as CronEntity;
+use Contao\CoreBundle\Repository\CronRepository;
+use Cron\CronExpression;
+use Psr\Log\LoggerInterface;
+
+class Cron
+{
+    /**
+     * @var string
+     */
+    public const SCOPE_WEB = 'web';
+
+    /**
+     * @var string
+     */
+    public const SCOPE_CLI = 'cli';
+
+    /**
+     * @var CronRepository
+     */
+    private $repository;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var array
+     */
+    private $crons = [];
+
+    public function __construct(CronRepository $repository, LoggerInterface $logger = null)
+    {
+        $this->logger = $logger;
+        $this->repository = $repository;
+    }
+
+    /**
+     * Add a cron service.
+     *
+     * @param string $interval The interval as a CRON expression
```

Comment:
why is this docblock needed?

Reply:
To provide additional info about the parameter (i.e. that you it is supposed to be a "CRON expression").